months.sql

```diff
@@ -0,0 +1,79 @@
+CREATE DATABASE witches;
```

Comment:
Your migrations should be creating and populating your tables, you should not have SQL code in your project.